===================================================================
RCS file: /cvs/ace/ace/games/mastermind.c,v
retrieving revision 1.5
retrieving revision 1.6
diff -p -u -r1.5 -r1.6
--- a/ace/games/mastermind.c	2001/09/21 19:22:51	1.5
+++ b/ace/games/mastermind.c	2003/02/11 18:35:59	1.6
@@ -210,7 +210,7 @@ key(int k, int x, int y)
   }
   if (k == 3 || k == 27 || k == 'q')
     exit(0);
-  if (k == KEY_F(2) || k == 'r')
+  if (k == KEY_F(2))
   {
     start_again();
     invalidate(0, 0, table_width, table_height);
